cmd/mirror_create.go

```diff
@@ -52,6 +52,9 @@ func aptlyMirrorCreate(cmd *commander.Command, args []string) error {
 	repo.SkipArchitectureCheck = context.Flags().Lookup("force-architectures").Value.Get().(bool)
 
 	if repo.Filter != "" {
+		if err != nil {
```

Comment:
what `err` is checked here ?

Reply:
seems useless code, removed

Reply:
@neolynx Yes, thanks for removing. I forgot to remove that while refactoring the code.